src/Uno.UI/UI/Xaml/Controls/MediaPlayer/MediaTransportControls.cs

```diff
@@ -207,12 +209,14 @@ protected override void OnApplyTemplate()
 			_timeRemainingElement = this.GetTemplateChild(TimeRemainingElementName) as TextBlock;
 
 			_progressSlider = this.GetTemplateChild(ProgressSliderName) as Slider;
+			PropertyChangedCallback test = OnSliderTemplateChanged;
```

Comment:
test?

Reply:
Updated ;) It was indeed a test, but I want to keep it